src/container/tabs.rs

```diff
@@ -106,22 +112,35 @@ impl Tabs {
         ui.painter()
             .rect_filled(ui.max_rect(), 0.0, behavior.tab_bar_color(ui.visuals()));
 
-        ui.with_layout(egui::Layout::right_to_left(egui::Align::Center), |ui| {
+        ui.with_layout(egui::Layout::left_to_right(egui::Align::Center), |ui| {
```

Comment:
No, there is a need for a `right_to_left` here so that users can add button to the left of the tabs, and for `egui_tiles` to show the `>` scroll button.

We should add an additional `left_to_right` layout to also allow users to add buttons to the left, and egui_tiles to add the `<` scroll button on the left